Tests/SwiftParserTest/ExpressionTests.swift

```diff
@@ -2225,6 +2225,49 @@ final class ExpressionTests: ParserTestCase {
       let _ = ((Int, Bool, String,) -> Void).self
       """
     )
+    
+    assertParse(
+      """
+      let _ = Array<(
+        bar: String,
+        baaz: String,
+      )>()
+      """,
+      substructure: FunctionCallExprSyntax(
```

Comment:
Had to include the `substructure:` because this test case succeeded as-is without the fix, even though the parsed output was incorrect:

```swift
assertParse(
  """
  let _ = Array<(
    bar: String,
    baaz: String,
  )>()
  """
)
```

Reply:
Nice, thank you!